### DISCUSSION TITLE
PvP polls and Season end

### DISCUSSION THREAD

**Discussion by balleny** [chromiecraft/chromiecraft#6691] [General]
A few questions related to the recent pvp polls and season end announcement:

(may at more questions at a later point, but would be for understanding purpose). These questions are specifically targeted at the chromiecraft server staff. If over people have more questions to add, please do so. 

1. Is there an ingame notification about the pvp poll?
2. Do you intent to make this vote available via the homepage too?
3. Why do you make this poll a workweek timeframe long only but exclude people that may be checking / playing on CC only during the weekend?
4. Do you intent to use the results for this seasons transition only or also for subsequent (tbc) arena seasons?
5. Why did you consider disabling 5s in the first place? Was someone doing some analysis checking how many people are playing 2s exclusively, 5s exclusively and how many played 2s and 5s to derive any impact on how much 2s and 5s bracket impacted each others activity? If so please link the details.
6. Why was there no voting option to disable 2s? 2s arena is by far the most imbalanced bracket, the bracket affected by the current bugs and class imbalances the most. Even on retail this was recognized. Rewards were removed from the 2s bracket and made accessible via 3s and 5s only.
7. Do you intent on correcting the arena item costs before season end, between the seasons, for the next season?
8. Why was an intermediate solution, giving access to s1 gear during s2 for reduced arena point costs, or a combination of arena points and honor, not considered for the voting?
9. As there is no arena monitor / game history afaik, do you review games conducted for potential wintrading and rewards distribution?
10. What ruleset to do you apply to be eligible for the season ends rewards? (some things typically include: a minimum amount of games played, a minimum amount of participation of the teams game, a maximum deviation of the personal rating to the teams rating, activity in the last x weeks of arena with at least y games per week, disqualifications for active buguse without reporting, ineligibility for pvp rewards because of previous pvp related banns/penalties of the same char/account/ip)
11. **Edit** Will battlegrounds tokens be removed, too?

**Comment by TotallyNotMyAccount** [comment]
Couldn't agree more on 5+6. Also, the stupid focus on the discord server annoys me as well.

**Comment by elthehablo** [comment]
1. No this was an oversight perhaps and could've been considered
2. No. We purposely try to keep specific things limited to 1 channel of communication. Polls have always been on Discord afaik
3. We feel a week is a long enough time to chime in. We assume people that are passionate about this will alert their guildmates/friends as well
The rest I don't have an answer for at this point

**Reply by balleny** [reply]
1. there was a regular ingame announcement
2. -
3. it was 5 days, Monday to Friday. The days with the highest activity (assumption by me), where excluded. the poll ended before saturday
